behat.yml.dist

```diff
@@ -24,9 +24,11 @@ default:
               priorities dropdown menu: "#nav-menu-expandable-group-3 .ecl-navigation-menu__links"
               search box: ".demo-block-header-search"
               sites switcher: ".demo-block-site-switcher"
+              link block list: ".ecl-link-block__list"
```

Comment:
This selector represent the actual dropdown content, so the proper name would be:
```
              dropdown content: ".ecl-link-block__list"
```
This will make it understandable  by business.